client/core/core.go

```diff
@@ -10755,3 +10755,59 @@ func (c *Core) checkEpochResolution(host string, mktID string) {
 
 	}
 }
+
+// RedeemGeocode redeems the provided game code with the wallet and redeems the
+// prepaid bond (code is a prepaid bond). If the user is not registered with
+// dex.decred.org yet, the dex will be added first.
+func (c *Core) RedeemGeocode(appPW, code []byte, msg string) (dex.Bytes, uint64, error) {
+	const dcrBipID = 42
```

Comment:
We should use the const from the network pkg? And say that this supports only dcr codes in the comment.